iceoryx_examples/icecubetray/README.md

```diff
@@ -0,0 +1,127 @@
+                                                 |          subscribe
+                                                 +---------------------------+
+                                                                             |
+                                                                             |
+                                                                             |
+                                                                            \ /
+                                 +-----------------------+
+                                 |                       |           +---------------------+
+                                 | Infotainment Shared   |           |   Display App       |
+                                 | Memory Segment        |  publish  |   @infotainment     |
+                                 |                       |  <------  |                     |
+                                 | r group: infotainment |           |            #        |
+                                 | w group: infotainment |           |           #         |
+                                 |                       |           |          #          |
+                                 |                       |           |     #   #           |
+                                 |                       |           |      # #            |
+                                 |                       |           +---------------------+
+                                 +-----------------------+
+```
+
+### RouDi and Apps
+
+#### Working setup
+
+RouDi is built with two static shared memory segments _infotainment_ and _privileged_. The access rights of the segments are configured as depicted in the graphic above.
+
+The `roudiConfig` is composed of a memory pool config called `mepooConfig`. When the segement is created, one needs to
+specific the reader group (first string), writer group (second string) as well as the `mepooConfig` (last parameter).
+
+```cpp
+iox::RouDiConfig_t roudiConfig;
+
+// Create Mempool Config
+iox::mepoo::MePooConfig mepooConfig;
+
+// We only send very small data, just one mempool per segment
+mepooConfig.addMemPool({128, 1000});
+
+/// Create an Entry for a new Shared Memory Segment from the MempoolConfig and add it to the RouDiConfig
+roudiConfig.m_sharedMemorySegments.push_back({"unprivileged", "privileged", mepooConfig});
+roudiConfig.m_sharedMemorySegments.push_back({"infotainment", "infotainment", mepooConfig});
+```
+
+The radar app is started with the user _perception_ and is sending data into the _privileged_ shared memory segment.
+
+The display app is started with the user _infotainment_. It reads the topic `{"Radar", "FrontLeft", "Object"}` from the privileged segement and forwards it as a slighty modified topic `{"Radar", "HMI-Display", "Object"}`. Because the user _infotainment_ only has write access to the infotainment segment, the data is written to this segment.
+
+!!! hint
```

Comment:
An additional hint would be good that the configuration can be done with TOML too.